elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -38,11 +38,6 @@ class PfeAccordion extends PFElement {
         type: String,
         alias: "disclosure",
         attr: "pfe-disclosure"
-      },
-      role: {
```

Comment:
I think we should leave the property and just remove the default and values

Reply:
We're not setting it in the component but that doesn't preclude someone else from having added role to their light DOM markup.

Reply:
I'm not sure we need to do that. The accordion component shouldn't have a role on the wrapper, yes, but I don't think we want to be policing which attributes people add to their light DOM. I just want to make sure that we're not adding a role. If a developer wants to add a role attribute, or any other attribute for that matter, I don't think we should stop them unless it's conflicting with the way the component needs to operate.